enclave/core/sgx/globals.c

```diff
@@ -213,8 +234,9 @@ const void* __oe_get_heap_base()
 size_t __oe_get_heap_size()
 {
 #ifdef OE_WITH_EXPERIMENTAL_EEID
-    if (oe_eeid)
-        return oe_eeid->size_settings.num_heap_pages * OE_PAGE_SIZE;
+    const oe_eeid_t* eeid = __oe_get_eeid();
+    if (eeid)
+        return eeid->size_settings.num_heap_pages * OE_PAGE_SIZE;
```

Comment:
For "static size" EEID enclave, the heap size info is in `oe_enclave_properties_sgx`.